docs/docs/jsx-in-depth.md

```diff
@@ -196,6 +196,14 @@ function NumberDescriber(props) {
 }
 ```
 
+Another option is to use ternary statements. For example, using the same function as above:
+
+```js{3-7}
+function NumberDescriber(props) {
+  return <div>{props.number} is an {props.number % 2 == 0 ? <strong>even</strong> : <i>odd</i>}
```

Comment:
You forgot to close the `</div>`.

I'm also not sure this section is fitting well here. This section is about the fact that statements can't be embedded in JSX. It is not about conditional rendering per se.

Maybe instead let's write something like

"You can learn more from more about [conditional rendering](https://facebook.github.io/react/docs/conditional-rendering.html) and [loops](https://facebook.github.io/react/docs/lists-and-keys.html) in the corresponding sections."